versioned_docs/version-3.x/modeling/relation.md

```
@@ -0,0 +1,284 @@
+enum ReferentialAction {
+    Cascade
+    Restrict
+    NoAction
+    SetNull
+    SetDefault
+}
+```
+
+-   `Cascade`
+
+    -   **onDelete**: deleting a referenced record will trigger the deletion of referencing record.
```

markdownlint-cli2 (0.17.2)

Unordered list indentation
Expected: 2; Actual: 4

(MD007, ul-indent)